px-k8s-postgres-all-in-one/assets/postgres-app-restore.yaml

```diff
@@ -9,6 +9,9 @@ spec:
       maxUnavailable: 1
     type: RollingUpdate
   replicas: 1
+  selector:
+    matchLabels:
+      app: postgres  
```

Comment:
This label should exist elsewhere in the app. as a metadata label or in the templat, often both.

```
The .spec.selector field defines how the Deployment finds which Pods to manage. In this case, you simply select a label that is defined in the Pod template (app: nginx). However, more sophisticated selection rules are possible, as long as the Pod template itself satisfies the rule.
``` 
-https://kubernetes.io/docs/concepts/workloads/controllers/deployment/